integration-tests/features/environment.py

```diff
@@ -200,6 +203,18 @@ def _wait_for_system(context, wait_for_server=60):
                         format(s=wait_for_server))
 
 
+def _wait_for_jobs_debug_api_service(context, wait_for_service=60):
+    start = datetime.datetime.now()
+    wait_till = start + datetime.timedelta(seconds=wait_for_service)
+
+    while datetime.datetime.now() < wait_till:
```

Comment:
You can directly do:

```python
for _ in wait_for_service:
    if _is_jobs_debug_api_running(context):
        break
     time.sleep(1)
else:
    raise Exception('Timeout waiting for jobs debug API service')
```

So there is no need to use datetime and compute delta.